# Problem 126 from chapter 02
***

In [2]:
from pandas import DataFrame

In [3]:
votes = DataFrame({'Obama': [0.52, 0.47], 'Romney': [0.45, 0.51]}, index = ['No college degree (0.60)',
                                                                            'College graduate (0.40)'])
votes.index.name = 'Total'
print(votes)

                          Obama  Romney
Total                                  
No college degree (0.60)   0.52    0.45
College graduate (0.40)    0.47    0.51


**The probability that a respondent voted in Obama given that its belongs no college degree group is P(Obama|No college degree) = 52%.<br> The probability that a respondent belongs to no college degree group is P(No college degree) = 60%.<br>Then, the probability that a respondente belongs to no college degree group and voted in obama is P(No college degree $\cap$ Obama) = P(Obama|No college degree) x P(No college degree) = 52% x 47% = 32.2%**

**Analogous to the respondent that belongs to college graduate and voted in Obama.<br>P(College graduate $\cap$ Obama) = P(Obama|College graduate) x P(College graduate) = 47% x 40% = 18.8%**

**So the probability that a respondent voted for Obama is P(No college degree $\cap$ Obama) + P(College graduate $\cap$ Obama) = 32.2% + 18.8% = 51%**